daemon/cmd/daemon_main.go

```diff
@@ -365,8 +365,13 @@ func initializeFlags() {
 	flags.Bool(option.BPFSocketLBHostnsOnly, false, "Skip socket LB for services when inside a pod namespace, in favor of service LB at the pod interface. Socket LB is still used when in the host namespace. Required by service mesh (e.g., Istio, Linkerd).")
 	option.BindEnv(option.BPFSocketLBHostnsOnly)
 
+	flags.Bool(option.EnableSocketLB, false, "Enable socket-based LB for E/W traffic")
+	option.BindEnv(option.EnableSocketLB)
```

Comment:
But why even add a new one? Can't we just hide/deprecate the existing one and then fold everything implicitly behind the KPR setting?

Reply:
My reasoning is that folks might still want to opt out from the socket LB. Removing the flag would make it impossible.